# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/general/browser.ini
# Commit: 802512ee57ae
# Full Hash: 802512ee57aec9d975ab078c48b9f37956012bc6
# Author: Neil Deakin <neil@mozilla.com>
# Date: 2015-04-17 19:45:37
# Regressor Bug: 1071562
# File Overlap Count: 1
# Description:
#   Bug 1071562, e10s, support non-text types in clipboard (html, images, etc), r=smaug
# ==============================================================================

diff -r ffd1b16f058b -r 802512ee57ae browser/base/content/test/general/browser.ini
--- a/browser/base/content/test/general/browser.ini	Thu Apr 16 01:11:05 2015 +0300
+++ b/browser/base/content/test/general/browser.ini	Thu Apr 16 15:38:12 2015 -0400
@@ -287,6 +287,7 @@
 [browser_bug1070778.js]
 [browser_canonizeURL.js]
 skip-if = e10s # Bug 1094510 - test hits the network in e10s mode only
+[browser_clipboard.js]
 [browser_contentAreaClick.js]
 [browser_contextSearchTabPosition.js]
 skip-if = os == "mac" || e10s # bug 967013; e10s: bug 1094761 - test hits the network in e10s, causing next test to crash